glean_parser/templates/ruby_server.jinja2

```diff
@@ -0,0 +1,138 @@
+{# The final Ruby code is autogenerated, but this
+Jinja2 template is not. Please file bugs! #}
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# AUTOGENERATED BY glean_parser v{{ parser_version }}. DO NOT EDIT.
+
+# frozen_string_literal: true
+
+# requires json, securerandom, and logger libs
+require 'json'
+require 'securerandom'
+require 'logger'
+require 'rbconfig'
+
+
+# this will be used for identifying logs that need to forward to Moz Data Pipeline
+GLEAN_EVENT_MOZLOG_TYPE = 'glean-server-event'
+
+module OS
+  def self.name
+    case RbConfig::CONFIG['host_os']
+    
+    when /linux/
+      'Linux'
+    when /darwin/
+      'OS X'
+    when /mswin|mingw32|windows/
+      'Windows'
+    when /solaris/
+      'Solaris'
+    when /bsd/
+      'BSD'
+    else
+      RbConfig::CONFIG['host_os']
+    end
+  end
+  def self.version
+    version = RbConfig::CONFIG['build_os'].match('\d{1,2}\.*\d{0,2}\.*\d{0,3}').to_s
+    if !version.nil?
+        version.to_s
```

Comment:
`.to_s` is already used in the match line above, so this is a no-op
```suggestion
        version
```

Reply:
also removing per above convos